src/foamadapter/solver/pimplefoam.py

```diff
@@ -0,0 +1,144 @@
+
+import pybFoam
+from pybFoam import (
+    Info,
+    Time,
+    Word,
+    adjustPhi,
+    computeCFLNumber,
+    constrainHbyA,
+    constrainPressure,
+    createPhi,
+    dictionary,
+    fvc,
+    fvm,
+    fvMesh,
+    fvScalarMatrix,
+    fvVectorMatrix,
+    pimpleControl,
+    setRefCell,
+    solve,
+    surfaceScalarField,
+    volScalarField,
+    volVectorField,
+)
+from pybFoam.turbulence import incompressibleTurbulenceModel, singlePhaseTransportModel
+
+
+class CFLNumber:
+
+    def __init__(self):
+        self.criteria = []
+        self.GREAT = 1e30
+        self.SMALL = 1e-15
+        self.maxDeltaT = 1e-3  # Default maximum deltaT
```

Comment:
I think this shouldn't be hardcoded. maybe pass as optional argument to __init__
```suggestion
        self.maxDeltaT = maxDeltaT  # Default maximum deltaT
```